### PR TITLE
Medieval Overhaul patch by Boldizsár

### PR DESCRIPTION
**Pull request by Safairette**
## Testing

Check tests you have performed:
- [ ] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [ ] Playtested a colony (specify how long)

### CODE REVIEW

**Comment by Breadbox2k19** [comment]
I suggest adding the nosetag into base CE patches, there are plenty of apparels that covers the nose but not the eye.

File: Patches/Medieval Overhaul/MO_Traits.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    </mods>
+    <match Class="PatchOperationSequence">
+      <operations>
+
+        <li Class="PatchOperationReplace">
+            <xpath>Defs/TraitDef[
+            defName="DankPyon_Fighter" or
+            defName="DankPyon_Veteran"]/commonality</xpath>
+            <value>
+              <commonality>0</commonality>
```
Comment (N7Huntsman) [inline Patches/Medieval Overhaul/MO_Traits.xml:16]: What's this?

File: Patches/Medieval Overhaul/MO_Materials.xml [hunk 2]
```diff
@@ -0,0 +1,620 @@
...
+				<xpath>Defs/ThingDef[defName="Leather_Plain"]/statBases/StuffPower_Armor_Blunt</xpath>
+				<value>
+					<StuffPower_Armor_Blunt>0.055</StuffPower_Armor_Blunt>
+				</value>
+			</li>
+        <!--Thick Leather-->
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Leather_Bear"]/statBases/StuffPower_Armor_Sharp</xpath>
+				<value>
+					<StuffPower_Armor_Sharp>0.12</StuffPower_Armor_Sharp>
```
Comment (N7Huntsman) [inline Patches/Medieval Overhaul/MO_Materials.xml:56]: I'm getting a duplicate error associated with this.

File: Patches/Medieval Overhaul/MO_Materials.xml [hunk 3]
```diff
@@ -0,0 +1,620 @@
...
+				<value>
+					<StuffPower_Armor_Blunt>0.06</StuffPower_Armor_Blunt>
+				</value>
+			</li>
+
+        <!--Thick Scale Leather-->
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Leather_Rhinoceros"]/statBases/StuffPower_Armor_Sharp</xpath>
+				<value>
+					<StuffPower_Armor_Sharp>0.14</StuffPower_Armor_Sharp>
```
Comment (N7Huntsman) [inline Patches/Medieval Overhaul/MO_Materials.xml:109]: I'm also getting an error associated with this operation.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
I can't get the ballista or trebuchet working. Additionally, the ballista bolts added by the mod can probably be removed.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
- Tavern Owner scenario needs ammo for bow.
- Chain mail helmets stats are a little too high. I'd make them slightly weaker than plate helmets.
- Stuffable helmets need Steeled instead of metallic. (Iron will probably need this tag added).
- Rox horn needs turned into a resource.
- Ballista bolts should be removed, since the turrets uses javelin for ammo.

**Comment by N7Huntsman** [comment]
I've copied the files and have made all the changes I noted above. For expediency, I'm closing this PR, and will make a new one with the updated files.

**Pull request closed without merging**